modules/ROOT/pages/glossary.adoc

```diff
@@ -15,69 +15,72 @@ endif::[]
 [[secta]]
 *A*
 
-Activity:: Role-based document transaction tracking (for send and receive) based on rules, actions, and functions.
+Acknowledgment:: Sent as a response to a B2B transmission. A TA1 acknowledgment serves as a receipt that a transmission was received. A functional acknowledgment includes details of the validation process.
+
+Activity:: A list of transmissions sent to and received from partners.
+
+Activity API:: Provides users with the ability to obtain transaction status.
+
+Anypoint Partner Manager:: Configures and manages integrations between EDI transmissions with your trading partners and your backend APIs; also tracks and reports on the transmission activity.
 
-Anypoint Partner Manager:: A single web-based user interface for Partner Configuration, Document flow  and Transaction Monitoring. The functionality that serves as a translation layer between internal systems and the various formats (X12, EDIFACT, XML, CSV, and others) and transport protocols (AS2, FTP/S, and others) commonly used in electronic transactions between companies.
 
-Activity API:: Provides users with the ability to obtain transaction status???
 
 [[secte]]
 *E*
 
-Endpoint:: Defines the protocol, address, and other details specific to an exchange of messages between partners such as Acknowledgement, Source Endpoint, and Target Endpoint.
+Endpoint: Source at host:: The origin of the message flow when defined by a host endpoint and message type, often an API.
+
+Endpoint: Target at host:: The final destination of a message flow when defined by a host endpoint and message type, often a Host API.
 
 [[sectg]]
 *G*
 
-Global Validation and Acknowledgement settings:: Default Host settings that apply to all partners. More here???
+Global Validation and Acknowledgement settings:: Settings that apply to all partners for message validation and acknowledgment responses.
 
 [[secth]]
 *H*
 
-Host:: The company that is subscribing to the Host in Partner Manager.
+Host:: The Anypoint customer that sends and receives transmissions with partners.
 
 [[secti]]
 *I*
 
-Identifier:: Uniquely identifies a particular Partner. Identifiers have a Type and Qualifier that indicate the identifier scheme and Value.
+Identifier:: Uniquely identifies a particular trading partner. For example, the DUNS number.
 
 [[sectm]]
 *M*
 
-Map:: Stores map type (XSLT or DataWeave) and scripting for transforming from one Message to another.
+Map:: A DataWeave script that transforms a B2B message.
```

Comment:
Please provide a link to a DataWeave document, such as an xref to https://docs.mulesoft.com/mule-runtime/4.1/transform-component-about -- As you can see from this link, DataWeave is also called the Transform Component, and should be identified using both terms.

Reply:
Noted for the main documentation.